### ISSUE TITLE
Commands do not work

### ISSUE THREAD

**Issue by SamMackrill** [nomadyow/Ocellus#5]
`2016-06-05 23:32 | ---------------------- Ocellus Plugin Initializing ----------------------
2016-06-05 23:32 | COMMAND:  press key bind
2016-06-05 23:32 | System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at OcellusPlugin.OcellusPlugin.VA_Invoke1(String context, Dictionary`2& state, Dictionary`2& shortIntValues, Dictionary`2& textValues, Dictionary`2& intValues, Dictionary`2& decimalValues, Dictionary`2& booleanValues, Dictionary`2& extendedValues)
2016-06-05 23:32 | 294467 systems in Index.
2016-06-05 23:33 | System.ArgumentException: The string was not recognized as a valid Uri.
Parameter name: inputUri
   at System.Xml.XmlReaderSettings.CreateReader(String inputUri, XmlParserContext inputContext)
   at System.Xml.XmlReader.Create(String inputUri, XmlReaderSettings settings, XmlParserContext inputContext)
   at System.Xml.Linq.XElement.Load(String uri, LoadOptions options)
   at System.Xml.Linq.XElement.Load(String uri)
   at OcellusPlugin.EliteBinds..ctor()
   at OcellusPlugin.OcellusPlugin.VA_Init1(Dictionary`2& state, Dictionary`2& shortIntValues, Dictionary`2& textValues, Dictionary`2& intValues, Dictionary`2& decimalValues, Dictionary`2& booleanValues, Dictionary`2& extendedValues)
2016-06-05 23:33 | COMMAND:  press key bind
2016-06-05 23:33 | System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at OcellusPlugin.OcellusPlugin.VA_Invoke1(String context, Dictionary`2& state, Dictionary`2& shortIntValues, Dictionary`2& textValues, Dictionary`2& intValues, Dictionary`2& decimalValues, Dictionary`2& booleanValues, Dictionary`2& extendedValues)
2016-06-05 23:47 | COMMAND:  get frontier credentials
2016-06-05 23:50 | Server response: OK
2016-06-05 23:50 | Server response: Redirect
2016-06-05 23:50 | Server response: OK
2016-06-05 23:50 | LOGIN OUTPUT:  <!DOCTYPE html>
<html>
<head>
        <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <title></title>
    <meta name="description" content="">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="robots" content="noindex, nofollow">

```
    <link rel="icon" href="https://companion.orerve.net:443/favicon.ico" type="image/x-icon">
</head>
```

<body class="">
        <hgroup class="panel-title">
    <h1>Verification Code</h1>
</hgroup>
    <form action="/user/confirm" method="post" class="form form-horizontal">
        <div class="errorSummary alert alert-danger">
        <h4>Please correct the following:</h4>
        <ul>
                                    <li>Missing required value</li>
                                </ul>
    </div>

```
<p>The verification code has now been sent to the email address associated with your Elite
    account.  Please enter the code into the box below.</p>

<div class="form-group has-error">
    <label for="username" class="col-sm-2 control-label">Confirmation code</label>
    <div class="col-sm-4">
        <input id="code" class="form-control" type="text" name="code" required autofocus>
    </div>
    <span class="help-block"></span>
</div>

<div class="form-group">
    <div class="col-sm-offset-2 col-sm-10">
        <button type="submit" class="btn btn-info">Confirm</button>
    </div>
</div>
```

</form>

```
    </body>
```

</html>

2016-06-05 23:50 | loginResponse:  verification
2016-06-05 23:50 | LoginResponse: verification
2016-06-05 23:50 | COMMAND:  get frontier verification
2016-06-05 23:50 | Server response: Redirect
2016-06-05 23:50 | Exception Caught:  System.Net.WebException: The remote server returned an error: (404) Not Found.
   at System.Net.HttpWebRequest.GetResponse()
   at Web.sendRequest(String url, CookieContainer cookieContainer, String referer, String sendData)
2016-06-05 23:50 | return code: False
2016-06-05 23:50 | return message: 
2016-06-05 23:50 | return HTML:  
2016-06-05 23:57 | COMMAND:  press key bind
2016-06-05 23:57 | System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at OcellusPlugin.OcellusPlugin.VA_Invoke1(String context, Dictionary`2& state, Dictionary`2& shortIntValues, Dictionary`2& textValues, Dictionary`2& intValues, Dictionary`2& decimalValues, Dictionary`2& booleanValues, Dictionary`2& extendedValues)
2016-06-05 23:57 | COMMAND:  check for upgrade
2016-06-05 23:57 | Server response: OK
2016-06-05 23:59 | COMMAND:  press key bind
2016-06-05 23:59 | System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at OcellusPlugin.OcellusPlugin.VA_Invoke1(String context, Dictionary`2& state, Dictionary`2& shortIntValues, Dictionary`2& textValues, Dictionary`2& intValues, Dictionary`2& decimalValues, Dictionary`2& booleanValues, Dictionary`2& extendedValues)
2016-06-06 00:00 | COMMAND:  press key bind
2016-06-06 00:00 | System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at OcellusPlugin.OcellusPlugin.VA_Invoke1(String context, Dictionary`2& state, Dictionary`2& shortIntValues, Dictionary`2& textValues, Dictionary`2& intValues, Dictionary`2& decimalValues, Dictionary`2& booleanValues, Dictionary`2& extendedValues)
2016-06-06 00:00 | COMMAND:  press key bind
2016-06-06 00:00 | System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at OcellusPlugin.OcellusPlugin.VA_Invoke1(String context, Dictionary`2& state, Dictionary`2& shortIntValues, Dictionary`2& textValues, Dictionary`2& intValues, Dictionary`2& decimalValues, Dictionary`2& booleanValues, Dictionary`2& extendedValues)
2016-06-06 00:02 | COMMAND:  press key bind
2016-06-06 00:02 | System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at OcellusPlugin.OcellusPlugin.VA_Invoke1(String context, Dictionary`2& state, Dictionary`2& shortIntValues, Dictionary`2& textValues, Dictionary`2& intValues, Dictionary`2& decimalValues, Dictionary`2& booleanValues, Dictionary`2& extendedValues)
2016-06-06 00:04 | COMMAND:  press key bind
2016-06-06 00:04 | System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at OcellusPlugin.OcellusPlugin.VA_Invoke1(String context, Dictionary`2& state, Dictionary`2& shortIntValues, Dictionary`2& textValues, Dictionary`2& intValues, Dictionary`2& decimalValues, Dictionary`2& booleanValues, Dictionary`2& extendedValues)
2016-06-06 00:04 | COMMAND:  press key bind
2016-06-06 00:04 | System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at OcellusPlugin.OcellusPlugin.VA_Invoke1(String context, Dictionary`2& state, Dictionary`2& shortIntValues, Dictionary`2& textValues, Dictionary`2& intValues, Dictionary`2& decimalValues, Dictionary`2& booleanValues, Dictionary`2& extendedValues)
2016-06-06 00:21 | COMMAND:  get debug`

**Comment by damonbrodie** [comment]
Can you email me your full debug.txt file to nomadyow@gmail.com, as well as the companion.json (if it exists in the same directory). 

I also just posted a patch that may have solved your issue along with others, you may want to try version 0.8.

**Comment by damonbrodie** [comment]
I believe this is fixed now in 0.9.
